nncf/torch/quantization/default_quantization.py

```diff
@@ -31,20 +31,11 @@
         operator_metatypes.PTDepthwiseConv3dSubtype,
         operator_metatypes.PTLinearMetatype,
         operator_metatypes.PTModuleLinearMetatype,
-        operator_metatypes.PTHardTanhMetatype,
```

Comment:
You should add a comment in-code with a link to the LPT docs and a brief explanation why we decided not to quantize single activations by default.

Reply:
Done, please check